lib/python/vdm/vdmrunner.py

```diff
@@ -100,25 +100,33 @@ def main():
   path = options[0]['filepath']
   server = options[0]['server']
 
+org_wd = os.getcwd()
+
 app_root = os.path.dirname(os.path.abspath(__file__))
 os.chdir(os.path.normpath(app_root))
 
 if path is None:
     home = expanduser("~")
-    # path = home + '.vdm' if home.endswith('/') else home + '/' + '.vdm'
-    path = os.path.join(home, '.vdm')
+    path = os.path.join(home, '.voltdb')
```

Comment:
Call these config_path and data_path so that it is clear to somebody reading the code.